kv/kvnemesis: TestApplier failed

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2169047&tab=buildLog) on [master@b7cf171ae2d77f3adababe7f42b2cfefc795a2ba](https://github.com/cockroachdb/cockroach/commits/b7cf171ae2d77f3adababe7f42b2cfefc795a2ba):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier565554319
--- FAIL: TestApplier (0.21s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier565554319
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:134: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:134
        	Error:      	Not equal: 
        	            	expected: "db1.AdminSplit(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db1.AdminSplit(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db1.AdminSplit(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2173273&tab=buildLog) on [master@71566fd4fa0a4efd579a2fb46c3738f92b5a4dbc](https://github.com/cockroachdb/cockroach/commits/71566fd4fa0a4efd579a2fb46c3738f92b5a4dbc):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier284648780
--- FAIL: TestApplier (0.20s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier284648780
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:136: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:136
        	Error:      	Not equal: 
        	            	expected: "db0.AdminMerge(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db0.AdminMerge(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db0.AdminMerge(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db0.AdminMerge(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2173275&tab=buildLog) on [master@71566fd4fa0a4efd579a2fb46c3738f92b5a4dbc](https://github.com/cockroachdb/cockroach/commits/71566fd4fa0a4efd579a2fb46c3738f92b5a4dbc):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier095989314
--- FAIL: TestApplier (1.16s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier095989314
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:134: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:134
        	Error:      	Not equal: 
        	            	expected: "db1.AdminSplit(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db1.AdminSplit(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db1.AdminSplit(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-race -parallel=2

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2174981&tab=buildLog) on [master@be8c1a17b940ecc56dc3521ab8264f662c88072c](https://github.com/cockroachdb/cockroach/commits/be8c1a17b940ecc56dc3521ab8264f662c88072c):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier533816228
--- FAIL: TestApplier (0.35s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier533816228
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:136: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:136
        	Error:      	Not equal: 
        	            	expected: "db0.AdminMerge(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db0.AdminMerge(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db0.AdminMerge(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db0.AdminMerge(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177381&tab=buildLog) on [master@457f1af5d19e0bcef9b5ea9e9125b3332717a86c](https://github.com/cockroachdb/cockroach/commits/457f1af5d19e0bcef9b5ea9e9125b3332717a86c):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier914831457
--- FAIL: TestApplier (0.16s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier914831457
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:134: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:134
        	Error:      	Not equal: 
        	            	expected: "db1.AdminSplit(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db1.AdminSplit(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db1.AdminSplit(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177382&tab=buildLog) on [master@457f1af5d19e0bcef9b5ea9e9125b3332717a86c](https://github.com/cockroachdb/cockroach/commits/457f1af5d19e0bcef9b5ea9e9125b3332717a86c):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier078998735
--- FAIL: TestApplier (1.18s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier078998735
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:134: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:134
        	Error:      	Not equal: 
        	            	expected: "db1.AdminSplit(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db1.AdminSplit(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db1.AdminSplit(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-race -parallel=2

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2179803&tab=buildLog) on [master@c3316a019b353f93b048fe800135ebf975fe653c](https://github.com/cockroachdb/cockroach/commits/c3316a019b353f93b048fe800135ebf975fe653c):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier041539482
--- FAIL: TestApplier (0.34s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier041539482
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:136: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:136
        	Error:      	Not equal: 
        	            	expected: "db0.AdminMerge(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db0.AdminMerge(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db0.AdminMerge(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db0.AdminMerge(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2181090&tab=buildLog) on [master@1049eaa26d983a0449049264d6c2ee2cc27454e2](https://github.com/cockroachdb/cockroach/commits/1049eaa26d983a0449049264d6c2ee2cc27454e2):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier885551818
--- FAIL: TestApplier (0.21s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier885551818
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:136: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:136
        	Error:      	Not equal: 
        	            	expected: "db0.AdminMerge(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db0.AdminMerge(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db0.AdminMerge(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db0.AdminMerge(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-parallel=4

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2181093&tab=buildLog) on [master@1049eaa26d983a0449049264d6c2ee2cc27454e2](https://github.com/cockroachdb/cockroach/commits/1049eaa26d983a0449049264d6c2ee2cc27454e2):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier297337764
--- FAIL: TestApplier (1.00s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier297337764
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:134: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:134
        	Error:      	Not equal: 
        	            	expected: "db1.AdminSplit(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db1.AdminSplit(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db1.AdminSplit(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- TAGS=
- GOFLAGS=-race -parallel=2

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(kv/kvnemesis).TestApplier failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2183081&tab=buildLog) on [master@41b82b264651968c9724df22b025c40ff68db398](https://github.com/cockroachdb/cockroach/commits/41b82b264651968c9724df22b025c40ff68db398):

```
=== RUN   TestApplier
test logs left over in: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier116365292
--- FAIL: TestApplier (0.37s)
    test_log_scope.go:154: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestApplier116365292
    test_log_scope.go:63: use -show-logs to present logs inline
    applier_test.go:134: 
        	Error Trace:	applier_test.go:51
        	            				applier_test.go:134
        	Error:      	Not equal: 
        	            	expected: "db1.AdminSplit(ctx, \"foo\") // aborted during DistSender.Send: context canceled"
        	            	actual  : "db1.AdminSplit(ctx, \"foo\") // aborted in distSender: context canceled"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled
        	            	+db1.AdminSplit(ctx, "foo") // aborted in distSender: context canceled
        	Test:       	TestApplier

```

<details><summary>More</summary><p>
Parameters:

- GOFLAGS=-json

```
make stressrace TESTS=TestApplier PKG=./pkg/kv/kvnemesis TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestApplier.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/kv/kvclient/kvcoord/dist_sender.go
+++ b/pkg/kv/kvclient/kvcoord/dist_sender.go
@@ -1577,7 +1577,7 @@ func (ds *DistSender) deduceRetryEarlyExitError(ctx context.Context) error {
 		return &roachpb.NodeUnavailableError{}
 	case <-ctx.Done():
 		// Happens when the client request is canceled.
-		return errors.Wrap(ctx.Err(), "aborted in distSender")
+		return errors.Wrap(ctx.Err(), "aborted in DistSender")
 	default:
 	}
 	return nil

--- a/pkg/kv/kvnemesis/applier_test.go
+++ b/pkg/kv/kvnemesis/applier_test.go
@@ -48,7 +48,12 @@ func TestApplier(t *testing.T) {
 		cancelledCtx, cancel := context.WithCancel(context.Background())
 		cancel()
 		require.NoError(t, a.Apply(cancelledCtx, &s))
-		assert.Equal(t, strings.TrimSpace(expected), strings.TrimSpace(s.String()))
+		actual := s.String()
+		// Trim out context canceled location, which can be non-deterministic.
+		// The wrapped string around the context canceled error depends on where
+		// the context cancellation was noticed.
+		actual = regexp.MustCompile(` aborted .*: context canceled`).ReplaceAllString(actual, ` context canceled`)
+		assert.Equal(t, strings.TrimSpace(expected), strings.TrimSpace(actual))
 	}
 
 	// Basic operations
@@ -63,9 +68,9 @@ func TestApplier(t *testing.T) {
 	check(t, step(get(`b`)), `db0.Get(ctx, "b") // ("2", nil)`)
 	check(t, step(scan(`a`, `c`)), `db1.Scan(ctx, "a", "c", 0) // (["a":"1", "b":"2"], nil)`)
 
-	checkErr(t, step(get(`a`)), `db0.Get(ctx, "a") // (nil, aborted during DistSender.Send: context canceled)`)
-	checkErr(t, step(put(`a`, `1`)), `db1.Put(ctx, "a", 1) // aborted during DistSender.Send: context canceled`)
-	checkErr(t, step(scanForUpdate(`a`, `c`)), `db0.ScanForUpdate(ctx, "a", "c", 0) // (nil, aborted during DistSender.Send: context canceled)`)
+	checkErr(t, step(get(`a`)), `db0.Get(ctx, "a") // (nil, context canceled)`)
+	checkErr(t, step(put(`a`, `1`)), `db1.Put(ctx, "a", 1) // context canceled`)
+	checkErr(t, step(scanForUpdate(`a`, `c`)), `db0.ScanForUpdate(ctx, "a", "c", 0) // (nil, context canceled)`)
 
 	// Batch
 	check(t, step(batch(put(`b`, `2`), get(`a`), scan(`a`, `c`))), `
@@ -80,10 +85,10 @@ func TestApplier(t *testing.T) {
 	checkErr(t, step(batch(put(`b`, `2`), get(`a`), scanForUpdate(`a`, `c`))), `
 {
   b := &Batch{}
-  b.Put(ctx, "b", 2) // aborted during DistSender.Send: context canceled
-  b.Get(ctx, "a") // (nil, aborted during DistSender.Send: context canceled)
-  b.ScanForUpdate(ctx, "a", "c") // (nil, aborted during DistSender.Send: context canceled)
-  db0.Run(ctx, b) // aborted during DistSender.Send: context canceled
+  b.Put(ctx, "b", 2) // context canceled
+  b.Get(ctx, "a") // (nil, context canceled)
+  b.ScanForUpdate(ctx, "a", "c") // (nil, context canceled)
+  db0.Run(ctx, b) // context canceled
 }
 `)
 
@@ -132,7 +137,7 @@ db0.Txn(ctx, func(ctx context.Context, txn *kv.Txn) error {
 	check(t, step(split(`foo`)), `db1.AdminSplit(ctx, "foo") // nil`)
 	check(t, step(merge(`foo`)), `db0.AdminMerge(ctx, "foo") // nil`)
 	checkErr(t, step(split(`foo`)),
-		`db1.AdminSplit(ctx, "foo") // aborted during DistSender.Send: context canceled`)
+		`db1.AdminSplit(ctx, "foo") // context canceled`)
 	checkErr(t, step(merge(`foo`)),
-		`db0.AdminMerge(ctx, "foo") // aborted during DistSender.Send: context canceled`)
+		`db0.AdminMerge(ctx, "foo") // context canceled`)
 }

